src/__integrationtests__/Attestation.spec.ts

```diff
@@ -0,0 +1,256 @@
+/**
+ * @group integration
+ */
+
+import {
+  faucet,
+  alice,
+  bob,
+  DriversLicense,
+  endow,
+  CtypeOnChain,
+  IsOfficialLicenseAuthority,
+} from './utils'
+import Claim from '../claim/Claim'
+import { IBlockchainApi } from '../blockchain/Blockchain'
+import getCached from '../blockchainApiConnection'
+import RequestForAttestation from '../requestforattestation/RequestForAttestation'
+import Attestation from '../attestation/Attestation'
+import AttestedClaim from '../attestedclaim/AttestedClaim'
+import { revoke } from '../attestation/Attestation.chain'
+import CType from '../ctype/CType'
+import ICType from '../types/CType'
+import { Identity } from '..'
+
+const attester = faucet
+const UncleSam = alice
+const claimer = bob
+
+describe('handling attestations that do not exist', () => {
+  xtest('Attestation.query', async () => {
+    return expect(Attestation.query('0x012012012')).resolves.toBeNull()
+  }, 30_000)
+
+  test('Attestation.revoke', async () => {
+    return expect(
+      Attestation.revoke('0x012012012', Identity.buildFromURI('//Alice'))
+    ).rejects.toThrow()
+  }, 30_000)
+})
+
+describe('When there is an attester, claimer and ctype drivers license', async () => {
+  beforeAll(async () => {
+    const ctypeExists = await CtypeOnChain(DriversLicense)
+    console.log(`ctype exists: ${ctypeExists}`)
+    console.log(`verify stored: ${await DriversLicense.verifyStored()}`)
+    if (!ctypeExists) {
+      await DriversLicense.store(attester)
+    }
+  }, 60_000)
+
+  it('should be possible to make a claim', () => {
+    const content = { name: 'Ralfi', age: 12 }
+    const claim = Claim.fromCTypeAndClaimContents(
+      DriversLicense,
+      content,
+      claimer.address
+    )
+    console.log(claim)
```

Comment:
can we have an assertion here instead of a log out?

Reply:
removing them all !